src/components/ContentNode/FigureCaption.vue

```diff
@@ -10,7 +10,7 @@
 
 <template>
   <figcaption class="caption">
-    <strong>{{ title }}</strong>&nbsp;<slot />
+    <strong v-if="title">{{ title }}</strong>&nbsp;<slot />
```

Comment:
Should this component and its `<figcaption>` wrapper ever even render if it has no text content to show?

Reply:
My idea was to still render a `<figcaption>` and populate it via the `<slot>`, I just `v-if` the bolded `title`, which this feature will not utilise.

Reply:
Oh, got it. I thought the title was all the text and missed the slot—I'm remembering that those are 2 separate things (title and content).